src/Compilers/CSharp/Test/Semantic/Semantics/NameOfTests.cs

```diff
@@ -1155,5 +1155,18 @@ public static void Main()
             var compilation = CreateCompilationWithMscorlib45(source);
             var comp = CompileAndVerify(source, expectedOutput: @"N1N2");
         }
+
+        [Fact, WorkItem(42, "github.com/dotnet/roslyn")]
```

Comment:
Should we use complete url as the second argument of WorkItem, i.e. github.com/dotnet/roslyn/issues/42?